redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java

```diff
@@ -251,7 +253,13 @@ protected RedisClientConfig createRedisConfig(NodeType type, RedisURI address, i
             String sslHostname) {
         RedisClientConfig result = super.createRedisConfig(type, address, timeout, commandTimeout, sslHostname);
         if (type == NodeType.SENTINEL && !usePassword) {
+            result.setUsername(null);
```

Comment:
why do you need to set the Username to null in both blocks?

Reply:
Sentinel doesnt understand usernames, it only accepts a password, if you try to pass in both, it will throw an invalid number of arguments error on the AUTH call..
so in either case with a node type of SENTINEL, we want to clear the username, (but not for other node types)

Reply:
Redis Sentinel supports username since 6.0 version. Please remove this line

Reply:
https://redis.io/topics/sentinel section `Configuring Sentinel instances with authentication`
`requirepass` configures how to authenticate to sentinel, but does not allow for a username as far as i can tell.

As of 6.0 sentinel can use username/passwords to connect to redis, so you could create a ACL user in redis and provide the credentials via `sentinel auth-pass` and `sentinel auth-user`

but when trying to use the auth command against a sentinel instance, it will reject multiple parameters:
```
redis-cli -p 26379
127.0.0.1:26379> auth username password
(error) ERR wrong number of arguments for 'auth' command
127.0.0.1:26379> auth password
(error) WRONGPASS invalid username-password pair
127.0.0.1:26379> auth a
OK
```
in that example, I configured a as the value in `requirepass` in sentinel.conf